docs/dev/table/sql/hints.zh.md

```diff
@@ -25,34 +25,25 @@ under the License.
 * This will be replaced by the TOC
 {:toc}
 
-SQL hints can be used with SQL statements to alter execution plans. This chapter explains how to use hints to force various approaches.
-
-Generally a hint can be used to:
-
-- Enforce planner: there's no perfect planner, so it makes sense to implement hints to
-allow user better control the execution;
-- Append meta data(or statistics): some statistics like “table index for scan” and
-“skew info of some shuffle keys” are somewhat dynamic for the query, it would be very
-convenient to config them with hints because our planning metadata from the planner is very
-often not that accurate;
-- Operator resource constraints: for many cases, we would give a default resource
-configuration for the execution operators, i.e. min parallelism or
-managed memory (resource consuming UDF) or special resource requirement (GPU or SSD disk)
-and so on, it would be very flexible to profile the resource with hints per query(instead of the Job).
-
-## Dynamic Table Options
-Dynamic table options allows to specify or override table options dynamically, different with static table options defined with SQL DDL or connect API, 
-these options can be specified flexibly in per-table scope within each query.
-
-Thus it is very suitable to use for the ad-hoc queries in interactive terminal, for example, in the SQL-CLI,
-you can specify to ignore the parse error for a CSV source just by adding a dynamic option `/*+ OPTIONS('csv.ignore-parse-errors'='true') */`.
-
-<b>Note:</b> Dynamic table options default is forbidden to use because it may change the semantics of the query.
-You need to set the config option `table.dynamic-table-options.enabled` to be `true` explicitly (default is false),
-See the <a href="{% link dev/table/config.zh.md %}">Configuration</a> for details on how to set up the config options.
-
-### Syntax
-In order to not break the SQL compatibility, we use the Oracle style SQL hint syntax:
+可以将 SQL 提示（hints）与 SQL 语句一起使用改变执行计划（execution plans）。本章介绍如何使用 SQL 提示增强各种方法。
```

Comment:
I think it's better to not translate the word `hints`. After I search with key words `sql hints` in baidu.com, I get more results about auto complete.

I think it's better to translate to 

```
SQL hints 是和SQL语句一起使用来改变执行计划的。
```